R/conditions.R

```diff
@@ -0,0 +1,54 @@
+
+subclass_index_errors <- function(expr) {
+  tryCatch(
+    expr,
+    vctrs_error_index_oob = function(cnd) {
```

Comment:
Why not catch `vctrs_error_index_oob_names ` separately?

Reply:
oh yes that makes more sense.